packages/stories/src/config/webpack/webpack.config.vendor.js

```diff
@@ -14,16 +14,24 @@ module.exports = ({ config, mode }) => {
     (p) => p.constructor.name === 'HtmlWebpackPlugin'
   );
 
-  const { templateParameters } = htmlWebpackPlugin.options;
-
-  htmlWebpackPlugin.options.templateParameters = (...args) => {
-    const htmlConfig = templateParameters(...args);
-
-    htmlConfig.files.js = [...vendorJsFiles, ...htmlConfig.files.js];
-    htmlConfig.files.css = [...vendorCssFiles, ...htmlConfig.files.css];
-
-    return htmlConfig;
-  };
+  htmlWebpackPlugin.options.templateParameters = (
+    compilation,
+    assets,
+    assetTags,
+    options
+  ) => ({
+    compilation,
+    webpackConfig: compilation.options,
+    htmlWebpackPlugin: {
+      tags: assetTags,
+      files: {
+        ...assets,
+        js: [...vendorJsFiles, ...assets.js],
+        css: [...vendorCssFiles, ...assets.css],
+      },
+      options,
+    },
+  });
```

Comment:
I think that it's more readable this way,
and also before, `templateParameters` wasn't a function so the whole build failed